client/client.go

```diff
@@ -81,6 +82,10 @@ type Client struct {
 	manualOverride bool
 }
 
+func redirectError(req *http.Request, via []*http.Request) error {
+	return errors.New("Sorry daemon could not process your request. Please make sure you entered correct arguments")
```

Comment:
@cpuguy83 is it OK if `docker inspect ../bad_name` prints `[]`? Because `inspect` sends a `GET` request.